js/background.js

```diff
@@ -1,29 +1,19 @@
 // callback to be called on click performed.
 function saveLink(info, tab){
-	// broadcast a message to the current tab
-	chrome.tabs.query({active: true}, function(tabs) {
-		// set message object to pass
-	    var message = { "message": "linqueue_get_anchor_title" };
-	    var key = getCurrentTime();
-	    var value = {};
-	    // send broadcast
-	    chrome.tabs.sendMessage(tabs[0].id, message, function(response){
-	    	// get the title from the response
-	    	var title = response && response.title;
-	    	// check if the title is present
-	    	if(!title){
-	    		title = tabs[0].title;;
-	    	}
-	    	// get the url
-	    	var url = info.linkUrl;
-	    	// set the title to value
-	    	value.title = title;
-	    	// set the url to value
-	    	value.url = url;
-	    	// save the url and title
-	    	saveData(key, value)
-	    });
-	});
+	var message = { "message": "linqueue_get_anchor_title" };
+	var key = getCurrentTime();
+	var value = {};
+
+	chrome.tabs.sendMessage(tab.id, message, function(response) {
+		//Title text got from content script
+		var title = (response && response.value)? response.value : "Title not found";
```

Comment:
I think it's good if you stick to the property of an object, like response should have a reasonable property like "response.title" 
The property value can be anything. Please change here.